inscriptions/ISic2832.xml

```diff
@@ -187,9 +187,9 @@
                     <lb n="1"/>
                     <gap reason="lost" extent="unknown" unit="character"/>καὶ Ἀ<unclear>γ</unclear><supplied reason="lost">άθη</supplied>
                     <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/>α · μνη<supplied reason="lost">σθῇ</supplied><gap reason="lost" unit="character" extent="unknown"/>
-                    <lb n="3"/><gap reason="lost" unit="character" extent="unknown"/><supplied reason="lost">ἐγὼ</supplied><unclear>ἐ</unclear>πικαλοῦμ<supplied reason="lost">αι τὸν λαὸν</supplied><gap reason="lost" unit="character" extent="unknown"/>
-                    <lb n="4"/><gap reason="lost" unit="character" extent="unknown"/><supplied reason="lost">ἐκ τ</supplied><unclear>η</unclear>͂ς ἐμῆς π<supplied reason="lost">ροικὸς βοηθεῖν</supplied><gap reason="lost" unit="character" extent="unknown"/>
-                    <lb n="5"/><gap reason="lost" unit="character" extent="unknown"/><supplied reason="lost">τοῖς</supplied><unclear>π</unclear>ένησι · κα<supplied reason="lost">ὶ ὀρφανοῖς (?) τῶν</supplied><gap reason="lost" unit="character" extent="unknown"/>
+                    <lb n="3"/><gap reason="lost" unit="character" extent="unknown"/><supplied reason="lost">ἐγὼ</supplied><unclear>ἐ</unclear>πικαλοῦμ<supplied reason="lost">αι</supplied> <supplied reason="lost">τὸν</supplied> <supplied reason="lost">λαὸν</supplied><gap reason="lost" unit="character" extent="unknown"/>
+                    <lb n="4"/><gap reason="lost" unit="character" extent="unknown"/><supplied reason="lost">ἐκ</supplied> <supplied reason="lost">τ</supplied><unclear>η</unclear>͂ς ἐμῆς π<supplied reason="lost">ροικὸς</supplied> <supplied reason="lost">βοηθεῖν</supplied><gap reason="lost" unit="character" extent="unknown"/>
+                    <lb n="5"/><gap reason="lost" unit="character" extent="unknown"/><supplied reason="lost">τοῖς</supplied><unclear>π</unclear>ένησι · κα<supplied reason="lost">ὶ</supplied> <supplied reason="lost">ὀρφανοῖς</supplied> (?) <supplied reason="lost">τῶν</supplied><gap reason="lost" unit="character" extent="unknown"/>
```

Comment:
Put a space after τοῖς and `@cert="low"` on the supplied with the question mark.

Reply:
Done. But...do you see something odd in the formatting on the second line?

Reply:
Uh oh, I did it too, let me see what it looks like now :-)